src/components/SingleArticleCard.jsx

```diff
@@ -39,7 +40,7 @@ const SingleArticleCard = ({ article, loading }) => {
               💬 {article.comment_count}
             </span>
           </Link>{" "}
-          <span className="article-text-smol">❤︎ {article.votes}</span>
+          <Votes />
```

Comment:
with regards to my previous comment, here is where you could pass the prop of `article={article}` to your votes component, which contains the data for the votes already

Reply:
Thank you very much. I have tried doing this but sometimes it causes the heart count to disappear upon refresh (until refreshed a second time), and I'm not quite sure why. I will put it back for now and work it out correctly later.